libacm/CMakeLists.txt

```diff
@@ -1,11 +1,17 @@
-set(HEADERS)
 set(CPPS
+  adecode.cpp
   aencode.cpp
-  adecode.cpp)
+)
 
 # these are the relevant source files from upstream libacm (https://github.com/markokr/libacm/)
 set(LIB_SRC
   decode.c
-  libacm.h)
+  libacm.h
```

Comment:
move into `third_party`?

Reply:
Splatted into `third_party/libacm` and `AudioEncode`.